### PR TITLE
[INJIWEB-43] added configuration changes for injiweb

### PR DESCRIPTION
**Pull request by challabeehyv**
added configuration changes for injiweb

### CODE REVIEW

File: esignet-insurance.properties [hunk 1]
```diff
@@ -488,9 +516,16 @@ mosip.esignet.authenticator.sunbird-rc.kba.entity-id-field=osid
...
 mosip.esignet.vciplugin.sunbird-rc.issue-credential-url=http://10.3.148.107/credential/credentials/issue
-mosip.esignet.vciplugin.sunbird-rc.supported-credential-types=InsuranceCredential
+mosip.esignet.vciplugin.sunbird-rc.supported-credential-types=LifeInsuranceCredential,InsuranceCredential
 mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
 mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/insurance-credential.json
 mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-get-url=http://10.3.148.107/registry/api/v1/Insurance/
 mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.cred-schema-id=did:schema:0d10a2cf-94de-4ffc-b32c-4f1a61ee05ba
 mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.cred-schema-version=1.0.0
+
+ntial-type.LifeInsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
```
Comment (swatigoel) [inline esignet-insurance.properties]: Looks like typo mistake

File: injiweb-demo-wellKnown.json [hunk 2]
```diff
@@ -76,7 +76,7 @@
           "name": "MOSIP Insurance",
           "locale": "en",
           "logo": {
-            "url": "https://api.qa-inji1.mosip.net/inji/mosip-logo.png",
+            "url": "https://api.collab.mosip.net/inji/mosip-logo.png",
```
Comment (swatigoel) [inline injiweb-demo-wellKnown.json]: Why is this collab url?

**Review by swatigoel — CHANGES_REQUESTED** [review]
Why are we still using demo issuer config?

**Comment by swatigoel** [comment]
@vishwa-vyom Please review this PR.

File: esignet-insurance.properties (hunk 1 above)
Comment (challabeehyv) [reply esignet-insurance.properties]: fixed

File: injiweb-demo-wellKnown.json (hunk 2 above)
Comment (challabeehyv) [reply injiweb-demo-wellKnown.json]: changed

**Comment by challabeehyv** [comment]
> Why are we still using demo issuer config?

existing dev environment was using this, removed it now as we are not using it anywhere

File: esignet-insurance.properties [hunk 3]
```diff
@@ -469,8 +469,36 @@ mosip.esignet.vci.key-values={\
...
+                              'policyExpiresOn': {'display': {{'name': 'Policy Expires On','locale': 'en'}}},\
+                              'policyName': {'display': {{'name': 'Policy Name','locale': 'en'}}},\
+                              'policyNumber': {'display': {{'name': 'Policy Number','locale': 'en'}}}\
+                       }},\
+                      'display': {{'name': 'Life Insurance Verifiable Credential', \
+                                    'locale': 'en', \
+                                    'logo': {'url': 'https://sunbird.org/images/sunbird-logo-new.png','alt_text': 'a square logo of a Sunbird'},\
+                                    'background_color': '#FDFAF9',\
+                                    'text_color': '#7C4616',\
+                                    'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'}\
```
Comment (vishwa-vyom) [inline esignet-insurance.properties]: I think the order needs to be outside display

File: esignet-insurance.properties [hunk 4]
```diff
@@ -469,8 +469,36 @@ mosip.esignet.vci.key-values={\
                                   'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'},\
                                   'background_color': '#FDFAF9',\
                                     'text_color': '#7C4616'}}\
-                 }}\
-        }\
+                 },\
+                  "LifeInsuranceCredential":{\
```
Comment (vishwa-vyom) [inline esignet-insurance.properties:501]: We need to update this under v11 also based on draft 11 spec and this section should be based on draft 12 spec.

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: esignet-insurance.properties (hunk 3 above)
Comment (challabeehyv) [reply esignet-insurance.properties]: changed this to outside of display

File: esignet-insurance.properties (hunk 4 above)
Comment (challabeehyv) [reply esignet-insurance.properties:501]: added under v11

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**